### PR TITLE
Proper Block Inscriptions HTML

### PR DESCRIPTION
**Pull request by veryordinally**
See #2329

### CODE REVIEW

**Comment by raphjaph** [comment]
Getting: `thread 'tokio-runtime-worker' panicked at 'index out of bounds: the len is 0 but the index is 36849', src/index/block_index.rs:177:25`

File: templates/block.html [hunk 1]
```diff
@@ -28,3 +28,11 @@ <h2>{{"Transaction".tally(self.block.txdata.len())}}</h2>
   <li><a href=/tx/{{txid}}>{{txid}}</a></li>
 %% }
 </ul>
+%% if self.inscriptions.len() > 0 {
```
Comment (raphjaph) [inline templates/block.html:31]: I think inscriptions should be above transactions. Maybe we should also preview them like on the inscriptions page

**Comment by veryordinally** [comment]
> Getting: `thread 'tokio-runtime-worker' panicked at 'index out of bounds: the len is 0 but the index is 36849', src/index/block_index.rs:177:25`

Improved error handling for when BlockIndex has not been initialized (happens on initial server startup, when index is initially catching up on new blocks). Message is now `Failed to get inscriptions in block: Block index not yet initialized`

**Review by raphjaph — APPROVED** [review]
Nice!

**Review by raphjaph — APPROVED** [review]
now its good

**Pull request merged**